src/smart-components/ChannelList/context/ChannelListProvider.tsx

```diff
@@ -314,8 +314,12 @@ function useChannelListContext(): ChannelListProviderInterface {
   return context;
 }
 
+const useChannelList = useChannelListContext;
+
 export {
   ChannelListProvider,
   // ChannelListContext,
+  // todo: rename all use{component}Context to use{component}
```

Comment:
https://sendbird.atlassian.net/browse/UIKIT-1624

Let me use the made ticket for this todo